crates/store/src/key/alias.rs

```diff
@@ -30,25 +27,39 @@ impl KeyComponent for Scope {
     type LEN = U32;
 }
 
+#[derive(Clone, Copy, Debug)]
+pub struct Alias;
+
 impl KeyComponent for Alias {
-    type LEN = U400;
+    type LEN = U50;
 }
 
 #[derive(Clone, Copy, Debug, Eq, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "borsh", derive(BorshSerialize, BorshDeserialize))]
 pub struct IdentityAlias(Key<(Kind, Scope, Alias)>);
 
 impl IdentityAlias {
-    pub fn new(kind: Kind, scope: [u8; 32], alias: Alias) -> Self {
+    fn create_key(kind: KindPrimitive, scope: [u8; 32], alias: AliasPrimitive) -> Self {
         let kind_array = GenericArray::<u8, U1>::from([kind as u8]);
         let scope_array = GenericArray::<u8, U32>::from(scope);
-
-        let mut alias_array = GenericArray::<u8, U400>::default();
+        let mut alias_array = GenericArray::<u8, U50>::default();
         let alias_str = alias.as_str();
         alias_array[..alias_str.len()].copy_from_slice(alias_str.as_bytes());
 
         Self(Key(kind_array.concat(scope_array).concat(alias_array)))
     }
+
+    pub fn context(alias: AliasPrimitive) -> Self {
+        Self::create_key(KindPrimitive::Context, [0u8; 32], alias)
+    }
+
+    pub fn application(alias: AliasPrimitive) -> Self {
+        Self::create_key(KindPrimitive::Application, [0u8; 32], alias)
+    }
+
+    pub fn identity(context_id: ContextId, alias: AliasPrimitive) -> Self {
+        Self::create_key(KindPrimitive::Identity, *context_id, alias)
+    }
```

Comment:
following convention with other keys, add convenience methods to retrieve the key components

```rust
fn kind(&self) -> KindPrimitive;
fn scope(&self) -> [u8; 32];
fn alias(&self) -> AliasPrimitive;
```